scripts/no-changes-in-high-risk-files.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+# To update the sha run shasum -a 256 ./Sources/Sentry/SentryNSURLSessionTaskSearch.m and copy the result in EXPECTED.
```

Comment:
@brustolin, I added this script, and it runs on CI. What do you think? If we have more files and change this often, we can add commands for updating the sha.

Reply:
I like the easy solution.
In the future, if we have many risk files we might rethink this script.